packages/manager/apps/web/client/app/hosting/cdn/shared/translations/Messages_fr_FR.json

```
@@ -1,14 +1,19 @@
 {
-  "hosting_cdn_shared_title": "Configurer votre CDN Basic",
-  "hosting_cdn_shared_info": "Retrouvez les paramètres de l’option Shared CDN. Des réglages standards sont appliqués par défaut pour vous permettre de démarrer simplement et rapidement.\nLes options avancées CDN Security et CDN Advance vous donnent accès à davantage de paramètres.",
+  "hosting_cdn_shared_title": "Configurer votre CDN {{range}} pour {{domainName}}",
+  "hosting_cdn_shared_info": "Retrouvez les paramètres de l’option Shared CDN. Des réglages standards sont appliqués par défaut pour vous permettre de démarrer simplement et rapidement.\nLes options CDN Security et CDN Advanced vous donnent accès à davantage de paramètres.",
```

i am not sure that escape char "\n" will be generate a break line.
Probably mistake on copy-past ?

```suggestion
  "hosting_cdn_shared_info": "Retrouvez les paramètres de l’option Shared CDN. Des réglages standards sont appliqués par défaut pour vous permettre de démarrer simplement et rapidement. Les options CDN Security et CDN Advanced vous donnent accès à davantage de paramètres.",
```

---

No copy paste here.
Also, I didn't change this part. I've just changed "Les options avancées CDN ..." to "Les options CDN..."